exporter/templates/cat1/r3/_result_value.xml

```diff
@@ -1,18 +1,18 @@
-{{ if not .EntrySection.IsValuesEmpty }}
-    {{ range .EntrySection.Entry.Values }}
-        {{ if .IsCodesPresent }}
+{{if not .EntrySection.IsValuesEmpty}}
+    {{range .EntrySection.Entry.Values}}
+        {{if .IsCodesPresent}}
 
-            {{ template "_result_value.xml" codeDisplayWithPreferredCode . .Coded.Codes "resultValue" }}
-        {{ else if ne .Scalar "" }}
-            {{ if eq .GetScalarType "num" }}
-                <value xsi:type="PQ" value="{{ .Scalar }}" {{ if ne .Units ""}}unit="{{ .Units }}"{{ else }}unit="1"{{ end }}/>
-            {{ else if eq .GetScalarType "bool" }}
-                <value xsi:type="BL" value="{{ .Scalar }}" />
-            {{ else }}
-                <value xsi:type="ST" >{{ .Scalar }}</value> 
-            {{ end }}
-        {{ end }}
-    {{ end }}
-{{ else }}
+            {{template "_code.xml" codeDisplayWithPreferredCode $.EntrySection.Entry .Coded "resultValue"}}
+        {{else if ne .Scalar ""}}
```

Comment:
Definitely makes sense. I was using HDS's layout by default but this makes far more sense structure-wise so I'm changing it to your recommendation.